INSTALL.rst

```diff
@@ -304,13 +304,52 @@ You should also set apache to start on boot::
 
   sudo chkconfig httpd on
 
-The networking server may be started by running::
+Running the network server:
+---------------------------
 
-  haas serve_networks &
 
-as the HaaS user. To make this happen on boot, add the following to ``/etc/rc.local``::
+A systemd script for running the network server is available in the 'scripts' directory.
+Name of the script is: haas_network.service
+
+Centos:
+------
+
+Centos uses systemd to controll all its processes. 
+
+Place the file haas_network.service under:
+``/usr/lib/systemd/system/``
```

Comment:
Generally systemd files can be placed at any of the two locations
1 . ``/usr/lib/systemd/system`` (Centos) or ``/lib/systemd/system/`` (Ubuntu)
2. ``/etc/systemd/system`` 

When ``systemctl enable <servicename>`` command is issued no matter where the file is placed among the above mentioned locations, it will create a symlink under 
``/etc/systemd/system/<some folder>``

Exact value of ``<some folder>`` is decided by what is the value of the directive
``WantedBy=`` in the service file. 

After reading up on various sources and 
https://wiki.archlinux.org/index.php/systemd#Writing_unit_files

I gathered that the general opinion is that location 1. is used by pacakges and location 2. is used by users to put their custom .service files.

I decided upon location 1. because HaaS is a package and will be made available as an
RPM or pip install in future.